ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/RemoveUnexpectedArgumentsFix.fs

```diff
@@ -0,0 +1,25 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+
+type RemoveUnexpectedArgumentsFix(warning: NotAFunctionError) =
+    inherit FSharpQuickFixBase()
+        
+    let notAFunctionExpr = warning.NotAFunctionExpr
+    let unexpectedArgs = warning.UnexpectedArgs
+    
+    override x.Text = "Remove unexpected arguments"
+
+    override x.IsAvailable _ =
+        isValid notAFunctionExpr && not (notAFunctionExpr :? ILiteralExpr)

```

Comment:
> `not (notAFunctionExpr :? ILiteralExpr)`

Why? Seems perfectly OK to remove the arg here:
<img width="221" alt="Screenshot 2020-02-12 at 14 29 24" src="https://user-images.githubusercontent.com/3923587/74331157-29d4ab00-4da4-11ea-8963-748e0a672ce9.png">